insQ.js

```diff
@@ -12,7 +12,10 @@ var insertionQ = (function () {
             strictlyNew: true,
             timeout: 20
         };
-
+    
+    if( options.timeout > 0 ) {
```

Comment:
I think you can just make this `const isTimeoutRequired = (options.timeout > 0 )`